src/Modules/NetworkSelectModal.tsx

```diff
@@ -8,10 +8,22 @@ import {
   ProgressBar,
   Typography,
 } from "@chainsafe/common-components";
+import styles from "../Constants/constants";
```

Comment:
Let's exclude redesign from this PR. We need to set up the current UI to support Cere -> Ethereum. Only minimal changes.

Reply:
I have introduced this small change in this PR because the current button hides while hovering the mouse over it.